src/main/java/io/lenses/connect/smt/header/InsertWallclockDateTimePart.java

```diff
@@ -81,7 +92,9 @@ public R apply(R r) {
       return null;
     }
 
-    final String value = valueExtractorF.apply(instantF.get());
+    Instant now = instantF.get();
+    final ZonedDateTime zonedDateTime = ZonedDateTime.ofInstant(now, timeZone);
+    final String value = valueExtractorF.apply(zonedDateTime);
```

Comment:
Surely it's a `value` but what does it mean? Maybe `extractedPartOfDate` ?
And possibly change `valueExtractorF` to `partOfDateExtractor` please?